packages/datastore/src/index.js

```diff
@@ -344,6 +344,20 @@ Datastore.prototype.double = Datastore.double = function(value) {
 };
 
 /**
+ * Helper function to check if something is a Datastore Double object.
+ *
+ * @param {*} something
+ * @return {boolean}
+ *
+ * @example
+ * datastore.isDouble(0.42); // false
+ * datastore.isDouble(datastore.double(0.42)); // true
+ */
+Datastore.prototype.isDouble = Datastore.isDouble = function(something) {
```

Comment:
Thank you for this whole PR, it is perfect! It's taken me so long to merge because I want to change the `something` var names with `value`. Otherwise, I think everything looks great 👍

Reply:
Sure thing, I'll change the argument names from `something` to `value`.